examples/app/app-bootstrap.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+angular.module('makeithappen', [
+    'ngRoute',
+    //'ngResource',
+    //'ui.ace',
+    'jsonforms-bootstrap'
```

Comment:
Instead of copying the whole app.js file we also could just include this line and make sure it is loaded after app.js in index-bootstrap.html